pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA      = "n/a"
 	name        = "cluster-operator"
 	source      = "https://github.com/giantswarm/cluster-operator"
-	version     = "3.1.2-dev"
```

Comment:
this only because of github action, it is still the right version 3.1.1 will the latest (previous version is 3.1.0)

Reply:
I do not follow what happened here. Why is this messed up now? Did the Github action do something or not? Did it do the right thing or not?

Reply:
it automatically creates a PR in case a new version got released. it creates a new PR with version '-dev' but it last time it was not doing the correct thing so it's showing the wrong version. 

The version which it wants to inject is the correct one it's 3.1.1

Reply:
`3.1.1` makes sense for the release. We just need to make sure that we get the dev version in after the release.